string[] fspset_path
---
geometry_msgs/PoseStamped[] fspset